### ISSUE TITLE
unknown layer type:  Lambda

### ISSUE THREAD

**Issue by XiujianLiu** [Dobiasd/frugally-deep#325]
i'm using u-net in my medical segmentation project, and need to convert input image from integer (0-255) to floating point (0.0-1.0). So after i defined the input layer i used a Lambda layer, here is my python code about the model:

  // input layer defining
  inputs = tf.keras.layers.Input((img_size, img_size, 1))

  // converting the input from integer to floating point
  // convert unit8 to float
  s = tf.keras.layers.Lambda(lambda x: x/255)(inputs)

  // going down... downscaling
  c1 = tf.keras.layers.Conv2D(32, (3, 3), activation='relu', kernel_initializer='he_normal', padding='same')(s)
  c1 = tf.keras.layers.Conv2D(32, (3, 3), activation='relu', kernel_initializer='he_normal', padding='same')(c1)
  p1 = tf.keras.layers.MaxPooling2D((2, 2))(c1)
  // and other layers.........

After i converted the .h5 file to .json file sucessfully, i load this file:
   const auto model = fdeep::load_model("fdeep_model.json");
and it throw an error:
![image](https://user-images.githubusercontent.com/89759319/156967088-1327250d-76be-4112-a83e-c6aa3933ac70.png)

it seems build the Lambda layer failed. How can i change the Lambda layer that frugally-deep can deal with? Is there a solution?

**Comment by Dobiasd** [comment]
Hi, and thanks for the question. I just added a section to the FAQ, explaining the situation: https://github.com/Dobiasd/frugally-deep/blob/master/FAQ.md#why-are-lambda-layers-not-supported
